packages/application-config/src/schema.ts

```diff
@@ -164,5 +164,10 @@ export interface JSONSchemaForCustomApplicationConfigurationFiles {
     }[];
     [k: string]: unknown;
   };
+  /**
+   * Sets the locales for which there are translations generated. Example: ["en", "de", "es"]
+   */
+  availableLocales?: string[];
```

Comment:
Also, the supported locales should be listed as enums, so that the config is correctly validated.

As for the name `availableLocales`, I'm not super convinced about this. I'll think about it.

Reply:
I updated the `schema.json` file with the new property (I'll change its name when the final one it's decided) and the tests for the `process-config` command.

Regarding the _enum_, the locales we will support are all of them, right? I mean, anyone creating a custom app can decide which locales they want to use within their app so I'm not sure how we can validate that with an enum.
Should we list every possible language or maybe we should stick to the ones defined for the `labelAllLocales`?

Reply:
> Regarding the enum, the locales we will support are all of them, right? 

No, there a specific number of locales that we support. 

```
['en', 'de', 'es', 'fr-FR', 'zh-CN', 'ja']
```

PS: we should probably document this a bit better in the new docs.